### PR TITLE
Add more filters for slots and schedule items in the admin view

### PR DESCRIPTION
**Pull request by drnlm**
This adds some f filters to the schedule admin site, to make finding particular slots and schedule items easier.

### CODE REVIEW

File: wafer/schedule/tests/test_admin.py [hunk 1]
```diff
@@ -181,24 +183,56 @@ def _make_filter(self, day):
...
+        self.assertEqual(len(lookups), 3)
+        self.assertEqual(lookups[0], ('11:00', '11:00'))
+        TestFilter = self._make_time_filter('12:00')
+        lookups2 = list(TestFilter.lookups(None, self.admin))
+        self.assertEqual(lookups, lookups2)
+
     def test_queryset_day_time(self):
         """Test queries with slots created purely by day + start_time"""
-        slots = {}
+        slots ={}
```
Comment (hodgestar) [inline wafer/schedule/tests/test_admin.py]: Missing space after `=`?

File: wafer/schedule/admin.py [hunk 2]
```diff
@@ -218,6 +218,134 @@ def validate_schedule():
     return errors
 
 
+# Useful filters for the admin forms
+
+class BaseDayFilter(admin.SimpleListFilter):
+    # Common logic for filtering on Slots and ScheduleItem.slots by Day
+    # We need to do this as a filter, since we can't use sorting since
+    # day is dynamic (either the model field or the previous_slot)
+    title = _('Day')
```
Comment (hodgestar) [inline wafer/schedule/admin.py]: Does each filter that inherits from this need a unique title? I'm just wondering how things will look in the admin interface?

Comment (drnlm) [reply wafer/schedule/admin.py]: Slots and ScheduleItems are different AdminForms so currently the children aren't ever on the same form. In this case the common name isn't a problem and adds a measure of consistency.

Comment (hodgestar) [reply wafer/schedule/admin.py]: Ah. Woot.

**Pull request merged**